azure-functions/acbs-function/acbs-http/index.js

```diff
@@ -10,3 +11,9 @@ const acbsHttp = async (context, req) => {
 };
 
 module.exports = acbsHttp;
+
+app.http('acbs-http', {
+  route: 'orchestrators/{orchestratorName}',
+  extraInputs: [df.input.durableClient()],
```

Comment:
This extra input was taken from [docs](https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-node-model-upgrade?tabs=nodejs-v3&pivots=programming-language-javascript#register-your-durable-client-input-binding) but its a subtly different example - the function.json differed in `orchestrationClient` vs `durableClient` so I need to find the correct replacement for the `extraInput` here

Reply:
I quite dislike having to add azure functions just for this function, but it seems needed.